DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/DiagnosticContentViewModelTests.cs

```diff
@@ -0,0 +1,240 @@
+                config,
+                diagnosticContent,
+                SelectedTutorials,
+                CustomisationId,
+                CentreId,
+                SectionId,
+                ProgressId,
+                CandidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.CourseTitle.Should().Be($"{applicationName} - {customisationName}");
+        }
+
+        [Test]
+        public void Diagnostic_content_should_have_section_name()
+        {
+            // Given
+            const string sectionName = "Section name";
+            var diagnosticContent = DiagnosticAssessmentHelper.CreateDefaultDiagnosticContent(
+                sectionName: sectionName
+            );
+
+            // When
+            var diagnosticContentViewModel = new DiagnosticContentViewModel(
+                config,
+                diagnosticContent,
+                SelectedTutorials,
+                CustomisationId,
+                CentreId,
+                SectionId,
+                ProgressId,
+                CandidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.SectionName.Should().Be(sectionName);
+        }
+
+        [Test]
+        public void Diagnostic_content_should_have_customisation_id()
+        {
+            // Given
+            const int customisationId = 11;
+            var diagnosticContent = DiagnosticAssessmentHelper.CreateDefaultDiagnosticContent();
+
+            // When
+            var diagnosticContentViewModel = new DiagnosticContentViewModel(
+                config,
+                diagnosticContent,
+                SelectedTutorials,
+                customisationId,
+                CentreId,
+                SectionId,
+                ProgressId,
+                CandidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.CustomisationId.Should().Be(customisationId);
+        }
+
+        [Test]
+        public void Diagnostic_content_should_have_section_id()
+        {
+            // Given
+            const int sectionId = 22;
+            var diagnosticContent = DiagnosticAssessmentHelper.CreateDefaultDiagnosticContent();
+
+            // When
+            var diagnosticContentViewModel = new DiagnosticContentViewModel(
+                config,
+                diagnosticContent,
+                SelectedTutorials,
+                CustomisationId,
+                CentreId,
+                sectionId,
+                ProgressId,
+                CandidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.SectionId.Should().Be(sectionId);
+        }
+
+        [Test]
+        public void Diagnostic_content_should_parse_html_url()
+        {
+            // Given
+            const int centreId = 11;
+            const int customisationId = 22;
+            const int candidateId = 33;
+            const int sectionId = 44;
```

Comment:
Any reason why you're defining these here when you've got them as class constants? 🙂